drush/default_content.drush.inc

```diff
@@ -34,9 +34,31 @@ function default_content_drush_command() {
     'required-arguments' => 2,
   ];
 
+  $items['default-content-bulk-export'] = [
+    'description' => dt('Bulk export for the whole content type entities.'),
+    'aliases' => ['dcebe'],
+    'arguments' => [
+      'entity_type' => dt('The entity type to export.'),
+    ],
+    'options' => [
+      'file' => dt('Write out the exported content to a file instead of stdout'),
+    ],
+  ];
+
+  $items['default-content-import'] = [
+    'description' => dt('Import content.'),
+    'aliases' => ['dcei']
+  ];
+
   return $items;
 }
 
+function drush_default_content_import() {
+
+  \Drupal::service('default_content.manager')->importContent('whatever');;
```

Comment:
I added a param to fulfill the method requirement, but it doesn't really need one. Please see DefaultContentManager.php L110